angular.json

```diff
@@ -28,8 +28,10 @@
               "src/assets"
             ],
             "styles": [
+              "./node_modules/bootstrap/dist/css/bootstrap.min.css",
+              "./node_modules/ngx-bootstrap/datepicker/bs-datepicker.css",
               "src/assets/styles/main.scss",
-              "node_modules/bootstrap/dist/scss/bootstrap.scss"
+              "node_modules/bootstrap/scss/bootstrap.scss"
```

Comment:
@Yuriy-27 do you really need these both lines?
"./node_modules/bootstrap/dist/css/bootstrap.min.css",
and
"node_modules/bootstrap/scss/bootstrap.scss"
??